packages/table/src/CheckboxCell/Checkbox.styles.tsx

```diff
@@ -0,0 +1,12 @@
+import { css } from '@leafygreen-ui/emotion';
+
+export const baseStyles = css`
+  // break the first-cell padding. would be cleaner to use a className here, but createUniqueClassname could have performance concerns.
+  margin-left: -16px;
+  width: 36px;
+`;
+
+export const headerStyles = css`
+  // break the first-cell padding. would be cleaner to use a className here, but createUniqueClassname could have performance concerns.
```

Comment:
why are we breaking the first-cell padding?

Reply:
Because the checkbox column needs to be skinner than other columns, but we DO want a base padding in a regular cell + `table-layout: fixed` as a default behavior. The checkbox cell needs to be passed as a child to Cell, so it's tricky to break the base Cell padding. I could explore an approach with a value created by `createUniqueClassname`, but I'm not 100% sure how that would work off the top of my head.